### ISSUE TITLE
Create a CU for RP for PR use case and start Titan testing

### ISSUE THREAD

**Issue by mturilli** [radical-collaboration/hpc-workflows#10]

**Comment by vivek-bala** [comment]
Create an example RP script: https://github.com/radical-cybertools/radical.ensemblemd/blob/usecase/seisflow/examples/usecase_seisflow/rp_example/forward_simulations_titan.py

Note issue https://github.com/radical-cybertools/radical.pilot/issues/1310

**Comment by vivek-bala** [comment]
Since that issue is fixed.

We can use the following script for Titan: https://github.com/radical-cybertools/radical.ensemblemd/blob/usecase/seisflow/examples/usecase_seisflow/rp_example/forward_simulations_titan.py

**Comment by mpbl** [comment]
Thanks!
A quick question: that's only for the mesher, right?

**Comment by vivek-bala** [comment]
Yes! I just tested the specfem, it started correctly but failed with a mpi error, expectedly. I am recompiling it against the new openmpi that the RP uses for the mpi support on titan.
